src/lib/__tests__/server-utilities.test.js

```diff
@@ -28,3 +30,45 @@ describe('md5Hash', () => {
     expect(fileNameHash).not.toBe(md5Hash(fileName));
   });
 });
+
+describe('compressJPEG', () => {
+  const filePath = './tmp/test.jpg';
+
+  beforeAll(() => {
+    if (!fs.existsSync('./tmp')) fs.mkdirSync('./tmp');
+
+    // Copy compressible JPEG to `./tmp` directory to simulate JPEG upload
+    return copyFile('./src/lib/__tests__/assets/compression-test.jpg', filePath);
+  });
+
+  /*
+   * We know compression-test.jpg will compress because it is a plain white image
+   * However, not all images will compress depending on the complexity of the images
+   */
+  it('compresses compressible JPEG', () => {
+    expect.assertions(1);
```

Comment:
Yep I see what you mean and this works well. So in the case that compressJPEG threw an error so `.then` wasn't called the test would still fail because of `expect.assertions`